### ISSUE TITLE
Readme screenshot issue in dark mode

### ISSUE THREAD

**Issue by finestructure** [SwiftPackageIndex/SwiftPackageIndex-Server#849]
I'm not sure there's much we can do in this case - the arrows in the graph are lost in the black background.

<img width="651" alt="CleanShot 2020-12-07 at 10 52 30@2x" src="https://user-images.githubusercontent.com/65520/101336445-c52a9580-387a-11eb-8970-b51ed89fb710.png">

Could we perhaps give images a light background by default so that when they're using an alpha channel they'd have the same background as on Github?

**Comment by finestructure** [comment]
Another option: render the whole Readme with a light background. That'd ensure it looks like on Github - what package authors are designing for. It might also serve as a nice separator and indicator what's the SPI preamble and what's the Readme. Not sure if that's great or desirable, just throwing it out there.

**Comment by daveverwer** [comment]
Or you could all use light mode. Best mode 😬

Another option, css can invert the colours in an image. That's not guaranteed to look good, especially if there's any photos in readme files, but I'd guess most images are diagram style. 

Screenshots would look pretty rough though, and there's lots of those.

**Comment by daveverwer** [comment]
I'll have a play with some options

**Comment by daveverwer** [comment]
I just saw this, but haven't looked at the details yet.

I wonder how GitHub deal with this issue! 

https://twitter.com/github/status/1336362679506784256?s=21

**Comment by Sherlouk** [comment]
![Captured 2020-12-08 at 7 15 58](https://user-images.githubusercontent.com/15193942/101530441-d57d6600-3989-11eb-8427-776a8dc0f71e.png)

tl;dr they don't 😂 
Their README background is slightly lighter which makes it a bit easier to see but yeah...

**Comment by finestructure** [comment]
They will have to now :D

https://twitter.com/github/status/1336362679506784256

**Comment by finestructure** [comment]
I guess I should have read your comments first 🤣

**Comment by daveverwer** [comment]
Like GitHub, we don't use a pure black for the background, we use a very dark blue.

I've just had a play with this and adding an artificial white background looks weird, and I don't think we should make any changes like that.

I think that GitHub (finally) adopting dark mode will make package authors more aware of this issue, and as they fix their packages for GitHub's dark mode, we'll inherit those fixed images too 👍

Closing for now.
